src/shared/components/DataComponent/DataQueryComponent.tsx

```diff
@@ -49,6 +49,15 @@ export const DataQueryComponent: FunctionComponent<DataQueryComponentProps> = ({
 					);
 				}
 
+				if (isEmpty(get(result, 'data'))) {
```

Comment:
Kan je deze niet mee in de if zetten waar `result.loading` gechecked wordt?

Reply:
nee want als er een error is, heeft die prioriteit